mindsdb/integrations/handlers/neuralforecast_handler/requirements.txt

```diff
@@ -1,2 +1,4 @@
 neuralforecast>=1.4.0, <2.0
-hyperopt<1.0
\ No newline at end of file
+hierarchicalforecast<1.0
+pytorch-lightning>=1.9.0, <2.0.0  # fixes an issue with the NF package
```

Comment:
As discussed elsewhere, this may introduce additional problems, maybe best to keep it out of this PR.

Reply:
Roger that, I'll roll this back.